sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -347,12 +347,6 @@ export interface CreateKeyOptions extends coreHttp.OperationOptions {
    * Size of the key
    */
   keySize?: number;
-  /**
-   * The retention dates of the softDelete data.
-   * The value should be >=7 and <=90 when softDelete enabled.
-   * **NOTE: This property will not be serialized. It can only be populated by the server.**
-   */
-  recoverableDays?: number;
```

Comment:
Why are you deleting it from keys?

Reply:
I used to have it on several of the option bags of several methods since I saw that they extend the key properties of the generated code. I decided to not add those since, as the comment says, as taken from the generated code, this property is not serialized, so it can't be used as an input. It is indeed added to the key properties though: https://github.com/Azure/azure-sdk-for-js/pull/7501/files#diff-91e6187853dc6c6c676058faaa81d8e2R202 if you check the latest diff against master, you will see no deletions.